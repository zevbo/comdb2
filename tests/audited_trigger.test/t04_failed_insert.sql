CREATE TABLE t(i int, s char(65))$$

CREATE LUA TRIGGER shoutout_to_monse ON (TABLE t FOR INSERT) AUDITED 

INSERT INTO t values(4)
INSERT INTO t values(5, "abc")

SELECT sleep(1)

SELECT * FROM "$audit_t" ORDER BY logtime

DROP TABLE t
DROP TABLE "$audit_t"
DROP LUA TRIGGER shoutout_to_monse