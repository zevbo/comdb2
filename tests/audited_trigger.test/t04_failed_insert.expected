[CREATE TABLE t(i int, s char(65))] rc 0
[CREATE LUA TRIGGER shoutout_to_monse ON (TABLE t FOR INSERT) AUDITED] rc 0
(rows inserted=1)
[INSERT INTO t values(5, "abc")] rc 0
(sleep(1)=1)
[SELECT sleep(1)] rc 0
(type='add', tbl='t', logtime="2021-07-05T114540.015 America/New_York", new_i=5, old_i=NULL, new_s='abc', old_s=NULL)
[SELECT * FROM "$audit_t" ORDER BY logtime] rc 0
[DROP TABLE t] rc 0
[DROP TABLE "$audit_t"] rc 0
[DROP LUA TRIGGER shoutout_to_monse] rc 0
